MapboxNavigation/ExitView.swift

```diff
@@ -141,4 +141,10 @@ class ExitView: StylableView {
         let labelTrailing = trailingAnchor.constraint(equalTo: exitNumberLabel.trailingAnchor, constant: 8)
         return [imageLeading, imageLabelSpacing, labelTrailing]
     }
+    
+    static func criticalHash(side: ExitSide, dataSource: DataSource) -> String {
```

Comment:
Still unclear on this.

Reply:
Yes -- the reason why it's called `criticalHash` is because the `side:` and `dataSource:` arguments contain critical values that affect the appearance of the `ExitView`. 

Considering that we're caching views by their rendered images, _any_ property that affects how an `ExitView` might appear needs to be accounted for in the cache key logic.